Wrappers/Python/examples/BoxWidgetAroundSlice.py

```diff
@@ -0,0 +1,17 @@
+import sys
+from PySide2 import QtWidgets
+from ccpi.viewer import viewer2D, viewer3D
+from SingleViewerCentralWidget import SingleViewerCenterWidget
+from ccpi.viewer.widgets.box_widgets import CreateBoxWidgetAroundSlice
+
+app = QtWidgets.QApplication(sys.argv)
+# can change the behaviour by setting which viewer you want
+# between viewer2D and viewer3D
+window = SingleViewerCenterWidget(viewer=viewer2D)
+line_widget = CreateBoxWidgetAroundSlice(window.frame.viewer, 'horizontal', 5, width=0)
```

Comment:
This could be actually a subclass called `LineWidgetOnSlice` or similar which will just set `width` to 0. 

In the use it will be clearer, rather than setting the width parameter to 0.